index.html

```diff
@@ -64,7 +64,7 @@ <h1>Welcome To Mother Tongues Network</h1>
     <section class="homepage-content">
       <img src="https://placehold.co/600x400" alt="" />
       <div class="home-text">
-        <h2>Mother Tongues Network (MTN)</h2>
+        <h2 class="sub-headings">Mother Tongues Network (MTN)</h2>
```

Comment:
You can probably target the H2 instead of adding all of these. We also need the styles on About and Contact.

Reply:
changed this to h2 only and removed class